apps/src/code-studio/initializeBlockPreview.js

```diff
@@ -0,0 +1,36 @@
+function update(blockSpace, container, editor) {
+  try {
+    // populate with new xml; because we do this every time the editor
+    // updates, there is the potential for many different kinds of
+    // errors here as we will end up updating as the levelbuilder is
+    // typing out the new content. Because of that and the fact that
+    // this is simply an informative but not functional view, we simply
+    // catch and ignore all errors.
+    var xml = $.parseXML(editor.getValue()).childNodes[0];
+    blockSpace.clear();
+    Blockly.Xml.domToBlockSpace(blockSpace, xml);
+  } catch (e) {
```

Comment:
That's what happens when the input is valid xml but not valid blockly blocks. Blockly itself doesn't have great error handling for that situation and will fail halfway through creating the invalid blocks, causing clear to complain when it tries to remove blocks that were never successfully created.